src/Service/SF1500/SF1500.php

```diff
@@ -116,27 +114,30 @@ public function getOrganisationFunktionerFromUserId(string $brugerId)
      */
     public function soegOrganisationFunktioner(?string $brugerId, ?string $funktionsNavn, ?string $organisationId, ?string $funktionsTypeId)
```

Comment:
Declare the return type for this and other functions in this class.

Reply:
Declared return types.